bcgov/House-Innovations-HOUSNAV
# HOUSNAV-187: initial build wizard work

[HOUSNAV-187](https://hous-bssb.atlassian.net/browse/HOUSNAV-187)

## Overview & Purpose
Create the build wizard page with initial functionality for selecting which walkthroughs to include. As noted in ticket, 'Begin Walkthrough' is hard coded to navigate the current 9.9.9 walkthrough.

## Summary of Changes
- Updated Building Type landing page to match design
- Moved javascript strings for some card content (building type analysis, single dwelling, and multi dwelling) into JSON that is loaded into the application for single source of some content
- Added more structure to json data naming and folders
- Added white colored link variant for the Button component
- Added secondary button variant for the Link component

## Testing
- Navigate to multi dwelling page.
- See default state with 'select all' enabled and 'begin walkthrough' disabled
- Select and deselect walkthroughs and notice updated state of application reflect available functionality

## Checklist
